ci/tests/general/general_alba.py

```diff
@@ -635,8 +635,9 @@ def filter_disks(disk_names, amount, disk_type):
         filtered_disks = dict()
         for node_id in disk_names.iterkeys():
             node_ids.append(node_id)
```

Comment:
I think you need to use a set io a list ... if we have 3 disks on 3 nodes, we'll have 9 entries in nodes_ids while we should have only 3